source/resource/designs/metal/basic.css

```diff
@@ -76,12 +76,16 @@ body br
   text-decoration:none;
 }
 
-.footer,
-.footer *
-{
-  color: #555;
+.footer, .footer {
```

Comment:
`.footer, .footer *` !== `.footer, .footer`

`.footer, .footer` should be simplified to `.footer`